examples/text-generation/run_generation.py

```diff
@@ -186,11 +186,20 @@ def main():
     parser.add_argument("--seed", type=int, default=42, help="random seed for initialization")
     parser.add_argument("--no_cuda", action="store_true", help="Avoid using CUDA when available")
     parser.add_argument("--num_return_sequences", type=int, default=1, help="The number of samples to generate.")
+    parser.add_argument(
+        "--fp16",
+        action="store_true",
+        help="Whether to use 16-bit (mixed) precision (through NVIDIA apex) instead of 32-bit",
+    )
     args = parser.parse_args()
 
     args.device = torch.device("cuda" if torch.cuda.is_available() and not args.no_cuda else "cpu")
     args.n_gpu = 0 if args.no_cuda else torch.cuda.device_count()
 
+    logger.warning(
```

Comment:
on line 213 you need to add code to actually use `args.fp16`
```python
if args.fp16: model.half()
```

Reply:
@sshleifer I have made this change in `run_generation.py`, But I have a doubt `run_glue.py` should also have this change right? 
But when I tried I got an error with a message as follows:
`msg = 'Found param distilbert.embeddings.word_embeddings.weight with type torch.cuda.HalfTensor, expected torch.cuda.FloatTe...alize, you do not need to call .half() on your model\nbefore passing it, no matter what optimization level you choose.'`

Reply:
you did the right thing!